src/fmgc/src/guidance/vnav/descent/AircraftToProfileRelation.ts

```diff
@@ -68,6 +68,13 @@ export class AircraftToDescentProfileRelation {
         return !this.topOfDescent || this.inertialDistanceAlongTrack.get() > this.topOfDescent.distanceFromStart;
     }
 
+    distanceToTopOfDescent(): number|null {
```

Comment:
I think we typically follow this format

```suggestion
    distanceToTopOfDescent(): number | null {
```